docs/Build-for-Edison-with-mraa.md

```diff
@@ -0,0 +1,80 @@
+# Build IoT.js with mraa for Intel Edison
+
+Edison board has Yocto Linux OS, so it is a enhanced case in [Build for Linux](https://github.com/Samsung/iotjs/wiki/Build-for-Linux).
+Edison board has mraa inside, so we can implement IoT.js's gpio module with mraa. See [#295](https://github.com/Samsung/iotjs/pull/295).
+
+## Overall steps
+
+1. Get the libvmraa.so and related headers in Edison's Yocto OS, and put them into the host PC (Ubuntu 14.04)
+2. Build the iotjs with mraa in the host PC
+3.  Copy the binary into the Edison Yocto OS.
+
+## Prerequisite
+
+host PC is Ubuntu 14.04
+mkdir a folder for the mraa related files.
+```
+cd harmony
```

Comment:
At this point it is not clear to me where is this 'harmony' folder. Could you add some explanation?